Solves skills not being deleted on character delete

L3nn0x opened the pull request:
solves #83

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/dev-osrose/osirose-new/178)
<!-- Reviewable:end -->

L3nn0x committed:
Update osirose.sql

L3nn0x committed:
Update osirose.sql

rminderhoud commented on Database/osirose.sql:
```diff
@@ -236,7 +236,8 @@ CREATE TABLE `skill` (
   `level` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  PRIMARY KEY (`char_id`,`id`)
+  PRIMARY KEY (`char_id`,`id`),
+  CONSTRAINT `char_id` FOREIGN KEY (`char_id`) REFERENCES `characters` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
```
Doesn't it have to be a constraint? Isn't it enough to declare it as foreign key?

rminderhoud replied:
Apparently this is valid syntax but redundant. I will accept and up to you if you want to change it or not

L3nn0x replied:
It's redundant but used in the whole file, so I'd go for consistency here.

L3nn0x replied:
Actually it poses problems so I'm removing it everywhere.

RavenX8 replied:
the reason why it's like that is because that is what the mysql workbench tool made it.

L3nn0x committed:
Update osirose.sql

If the CONSTRAINT symbol clause is given, the symbol value, if used, must be unique in the database. A duplicate symbol will result in an error similar to: ERROR 1022 (2300): Can't write; duplicate key in table '#sql- 464_1'. If the clause is not given, or a symbol is not included following the CONSTRAINT keyword, a name for the constraint is created automatically.

rminderhoud commented:
LGTM
